gms/gossip_digest_syn.cc

```diff
@@ -9,17 +9,15 @@
  */
 
 #include "gms/gossip_digest_syn.hh"
-#include <ostream>
 
-namespace gms {
-
-std::ostream& operator<<(std::ostream& os, const gossip_digest_syn& syn) {
-    os << "cluster_id:" << syn._cluster_id << ",partioner:" << syn._partioner << ",group0_id:" << syn._group0_id << ",";
-    os << "digests:{";
+auto fmt::formatter<gms::gossip_digest_syn>::format(const gms::gossip_digest_syn& syn, fmt::format_context& ctx) const
+        -> decltype(ctx.out()) {
+    auto out = ctx.out();
+    // out = fmt::format_to(out, "cluster_id:{},partioner:{},group0_id{},"
+    //                      syn._cluster_id, syn._partioner, syn._group0_id);
```

Comment:
Hi @tchaikov, why did you commented this part out?

Reply:
it was a mistake. i will bring it back with a pull request tomorrow. and will add you as a reviewer. thanks for pinging me on this.

Reply:
created https://github.com/scylladb/scylladb/issues/23142